src/Sql/dbo/Stored Procedures/OrganizationPasswordManager_Create.sql

```diff
@@ -0,0 +1,58 @@
+CREATE PROCEDURE [dbo].[OrganizationPasswordManager_Create]
+    @Id UNIQUEIDENTIFIER OUTPUT,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Plan NVARCHAR(50),
+    @PlanType TINYINT,
+    @Seats INT,
+    @UseTotp BIT,
+    @UsersGetPremium BIT,
+    @Storage BIGINT,
+    @MaxStorageGb SMALLINT,
+    @MaxAutoscaleSeats INT
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    INSERT INTO [dbo].[OrganizationPasswordManager]
+    (
+        [Id],
+        [OrganizationId],
+        [Plan],
+        [PlanType],
+        [Seats],
+        [UseTotp],
+        [UsersGetPremium],
+        [Storage],
+        [MaxStorageGb],
+        [MaxAutoscaleSeats],
+        [RevisionDate]
+    )
+    VALUES
+    (
+        @Id,
+        @OrganizationId,
+        @Plan,
+        @PlanType,
+        @Seats,
+        @UseTotp,
+        @UsersGetPremium,
+        @Storage,
+        @MaxStorageGb,
+        @MaxAutoscaleSeats,
+        GETUTCDATE()
+    )
+
+    UPDATE [dbo].[Organization]
+    SET
+        [Plan] = @Plan,
+        [PlanType] = @PlanType,
+        [Seats] = @Seats,
+        [UseTotp] = @UseTotp,
+        [UsersGetPremium] = @UsersGetPremium,
+        [Storage] = @Storage,
+        [MaxStorageGb] = @MaxStorageGb,
+        [MaxAutoscaleSeats] = @MaxAutoscaleSeats,
+        [RevisionDate] = GETUTCDATE()
+    WHERE Id = @OrganizationId
```

Comment:
I'm confused by this. It seems like you're wanting to sync up Organization and OrganizationPasswordManager, but shouldn't this be an insert?

Reply:
It also looks like you aren't creating OPMs on Organization_Create.

this needs to work in both directions: old -> new server and new -> old server without data loss.

Reply:
It should be an update. This keeps PasswordManager in sync with Organizations. Although we should add an upsert to Organizations to create this table if any of those values are changed. (The new server should omit those fields to ensure it doesn't trigger that pathway.